prime-router/settings/organizations-local.yml

```diff
@@ -332,10 +334,13 @@
         translation:
           type: HL7
           useBatchHeaders: true
+          receivingOrganization: elr
           receivingApplicationName: NMDOH
           receivingApplicationOID: 2.16.840.1.113883.3.5364
           receivingFacilityName: NMDOH
           receivingFacilityOID: 2.16.840.1.113883.3.5364
+          truncateHDNamespaceIds: true
+          nameFormat: APHL_LIGHT
         timing:
```

Comment:
A new name format! No seriously, I have an idea on how to replace this with a more programmatic option, but I'm not there yet.

Reply:
Yes.   We need to assume there will be a zillion name formats.